### PR TITLE
Added content : Galactic Center

### PR DESCRIPTION
**Pull request by elitemn**
Added a new POI named "Galactic Center" with new associated views based on the _Solar System_ view.  Added voice overs and commentary for the new views. Please review and let me know what you think.

### CODE REVIEW

File: Assets/SolarSystem/Planets/LODs/Planet_LODs.mat [hunk 1]
```diff
@@ -336,7 +336,7 @@ Material:
       second: {r: 1, g: 0, b: 0, a: 0}
     - first:
         name: _SunPosition
-      second: {r: -0.019818218, g: -0.09801179, b: 1.495989, a: 0}
```
Comment (timGerken) [inline Assets/SolarSystem/Planets/LODs/Planet_LODs.mat]: This change doesn't seem to have anything to do with your PR and is probably a side effect of Unity's editor. There are pending changes in the Phone branch that will help with Unity's editor and material modification. Please remove this file from your PR.

File: Assets/SolarSystem/Planets/EarthClouds.mat [hunk 2]
```diff
@@ -82,7 +82,7 @@ Material:
       second:
         m_Texture: {fileID: 2800000, guid: b51ecab21caa2a1458f8cbc1d379f16f, type: 3}
         m_Scale: {x: 2, y: 2}
-        m_Offset: {x: 0.04606036, y: 0}
```
Comment (timGerken) [inline Assets/SolarSystem/Planets/EarthClouds.mat]: This change doesn't seem to have anything to do with your PR and is probably a side effect of Unity's editor. There are pending changes in the Phone branch that will help with Unity's editor and material modification. Please remove this file from your PR.

File: Assets/SolarSystem/OrbitalTrails/OrbitalTrail.mat [hunk 3]
```diff
@@ -93,7 +93,7 @@ Material:
       second: 0.04
     - first:
         name: _GlobalScale
-      second: 10.079262
```
Comment (timGerken) [inline Assets/SolarSystem/OrbitalTrails/OrbitalTrail.mat]: This change doesn't seem to have anything to do with your PR and is probably a side effect of Unity's editor. There are pending changes in the Phone branch that will help with Unity's editor and material modification. Please remove this file from your PR.

File: Assets/SolarSystem/Asteroids/AsteroidMat.mat [hunk 4]
```diff
@@ -130,4 +130,4 @@ Material:
       second: {r: 0.43, g: 0.43, b: 0.43, a: 1}
     - first:
         name: _LightPosition
-      second: {r: -0.019818218, g: -0.09801179, b: 1.495989, a: 0}
```
Comment (timGerken) [inline Assets/SolarSystem/Asteroids/AsteroidMat.mat]: This change doesn't seem to have anything to do with your PR and is probably a side effect of Unity's editor. There are pending changes in the Phone branch that will help with Unity's editor and material modification. Please remove this file from your PR.

File: Assets/Galaxy/GalaxyPlaneMat.mat [hunk 5]
```diff
@@ -197,7 +197,7 @@ Material:
       second: 1
     - first:
         name: _TransitionAlpha
-      second: 0.000013768673
```
Comment (timGerken) [inline Assets/Galaxy/GalaxyPlaneMat.mat]: This change doesn't seem to have anything to do with your PR and is probably a side effect of Unity's editor. There are pending changes in the Phone branch that will help with Unity's editor and material modification. Please remove this file from your PR.

**Comment by timGerken** [comment]
@elitemn it looks like this is the beginning of some pretty cool new content!

Some things I noticed:

When in Galaxy view:
1. On the POI card for Galactic Center, the subtext "Sagittarius A" is clipped/illegible

When in GalaxyCenter view:
1. S102 does not appear to exactly follow its orbit
2. S102_LOD1 seems to be way too big and orbits with S102 like a massive halo
3. S102's speed combined with its color causes color separation (red/green) during most of its orbit.
4. S2's orbit is "very pointy" where it is nearest Sagittarius A*
5. On the POI card for Sagittarius A*, the subtext "Complex Radio Source (Supermassive Black Hole)" is clipped/illegible
6. The True-scale POI should be removed as its VO is still that for the Solar System
7. The lines from the POI cards for both S2 and S102 do not touch their orbits (as they do in SolarSystem view)

File: Assets/SolarSystem/Planets/LODs/Planet_LODs.mat (hunk 1 above)
Comment (elitemn) [inline Assets/SolarSystem/Planets/LODs/Planet_LODs.mat]: Thanks Tim, not sure why those were included. Do you have a preferred method to do this? (revert commit? some git magic I am not aware of?)

Comment (timGerken) [inline Assets/SolarSystem/Planets/LODs/Planet_LODs.mat]: I don't have a preferred method nor am I anywhere near a git expert :). One recommendation from a teammate is to commit the original versions of those files and update your PR to include that commit as well.

**Comment by elitemn** [comment]
@timGerken  Thanks for the great feedback! I'll dig in and see what I can learn.

File: Assets/SolarSystem/Planets/EarthClouds.mat.bak [hunk 6]
```diff
@@ -0,0 +1,276 @@
+%YAML 1.1
```
Comment (timGerken) [inline Assets/SolarSystem/Planets/EarthClouds.mat.bak]: not sure you intended to add this file...

File: Assets/GalacticCenter/Sun/SunGlowCamDistSetter.cs [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+        {
+            DistanceFromCamera = (Camera.main.transform.position - transform.position).magnitude;
+
+            float distanceA = ColorADistanceFromDistance.Evaluate(DistanceFromCamera);
+            float smoothnessA = 1 / ColorASmoothnessFromDistance.Evaluate(DistanceFromCamera);
+
+            colorAParams = mat.GetVector("_ColorAParams");
+            colorAParams.x = distanceA;
+            colorAParams.y = smoothnessA;
+            mat.SetVector("_ColorAParams", colorAParams);
```
Comment (timGerken) [inline Assets/GalacticCenter/Sun/SunGlowCamDistSetter.cs]: Unity's editor will persist these changes to disk. Recommendation is to cache the original value in Start() and restore it in OnDestroy(). There are similar pending changes in the phone branch.

Comment (elitemn) [inline Assets/GalacticCenter/Sun/SunGlowCamDistSetter.cs]: Thanks Tim. I’ll implement your solution shortly. Good to know that is what’s going on.  I’ll take a close look at the other changes you implemented in the phone branch as well.

File: Assets/SolarSystem/Planets/EarthClouds.mat.bak (hunk 6 above)
Comment (elitemn) [inline Assets/SolarSystem/Planets/EarthClouds.mat.bak]: Good catch.

**Comment by timGerken** [comment]
Looking better!

Some things I noticed:
- Experimentally, a value of 400 seems to work better (makes for a smoother ellipse) for Step Count in the trails for S2 and S102.
- It looks like a lot of inadvertent .mat changes snuck into this commit. Unity's play-mode will persist dynamic changes to materials. I'm sure there is a git way to solve this (probably some combination of rebaseing and something(s) else).

**Comment by timGerken** [comment]
One more thing:
- The ratio between the periods of S2 and S102 seems to be off a bit (if [Wikipedia](https://en.wikipedia.org/wiki/Sagittarius_A*) is to be believed). Currently the ratio is .666 (Period Real is .719) and both should be closer to .754.

**Comment by elitemn** [comment]
I get a ratio of .71875 (.719) according to this research supplement from UCLA: [Supplement] (http://www.astro.ucla.edu/~ghezgroup/gc/research/1225506s.pdf) (Table S4 page 10). I believe this measurement is newer and more precise than the Wikipedia entry (2002) because I find reference to measurements recorded in 2011 (page 3, last paragraph).  I did round the orbital period from 16.17 years to 16 years in my calculation.  If we are going to be that precise, I suggest we change to .712 (11.5/16.17). What are your thoughts?

**Comment by timGerken** [comment]
I'm in favor of the more current research :)
Go ahead and update both Period (what is currently used in the scene) and Period Real (what would be used if "Realistic View" were implemented) to reflect that change.

**Comment by timGerken** [comment]
Hi @elitemn, thanks for making all of these updates! There are two more things I noticed today, though I think we are getting close.

- Sagittarius A*'s LOD when it is very small looks like it is still using that of the Sun (from the SolarSystem view). This means that when scaled down (or viewed from "very far away") it no longer has the black-hole-sun feel to it.
- Ideally, because their orbits are so large at the default scale, S2 and S102 should start orbiting closer to their POI markers. That will make it easier for the user to connect each POI to the Star's orbit it is marking.

Keep up the great work!

**Comment by elitemn** [comment]
Thanks again Tim for the keen eye.  I will have to think about the starting positions of the galactic center stars.  These positions are calculated based off the orbital values entered and the current time (from what I understand at this point).  Perhaps instead of messing with the orbital mechanics, I could adjust the starting zoom level to be more appropriate.

File: Assets/GalacticCenter/Sun/Sun_LensFlareCard.mat [hunk 8]
```diff
@@ -151,7 +151,7 @@ Material:
       second: {r: 9, g: 14, b: 0, a: 0}
     - first:
         name: _Tint
-      second: {r: 0.6967206, g: 0.13831954, b: 0.21919131, a: 1}
```
Comment (timGerken) [inline Assets/GalacticCenter/Sun/Sun_LensFlareCard.mat]: Looks good! Thanks!

File: Assets/GalacticCenter/Sun/Sun_LensFlareCard.mat [hunk 9]
```diff
@@ -151,7 +151,7 @@ Material:
...
+      second: {r: 0.022058845, g: 0.022058845, b: 0.022058845, a: 1}
```
Comment (timGerken) [inline Assets/GalacticCenter/Sun/Sun_LensFlareCard.mat]: Looks good, thanks!

**Comment by timGerken** [comment]
Another option, though a bit more complex, would be to add a directional indicator to the POI marker that points to where to actually find the sun/planet.

That's additional functionality that is outside of the scope of this PR. It would make more sense to start with changing the default scale of the Sagittarius A* scene and do the POI/DI implementation as a separate PR.

**Comment by timGerken** [comment]
love it!

**Pull request merged**